ext/dapr-ext-grpc/dapr/ext/grpc/app.py

```diff
@@ -108,7 +108,7 @@ def decorator(func):
         return decorator
 
     def subscribe(self, pubsub_name: str, topic: str, metadata: Optional[Dict[str, str]] = {},
-                  rule: Optional[Rule] = None):
+                  rule: Optional[Rule] = None, disable_topic_validation = False):
```

Comment:
Please use the Optional type for this parameter.

```suggestion
                  rule: Optional[Rule] = None, disable_topic_validation: Optional[bool] = False):
```

Reply:
thx for the review, I applied your suggestion